js/get-slider.js

```diff
@@ -0,0 +1,43 @@
+const slider = document.querySelector('.ad-form__slider');
+const MIN_VALUE_PRICE = 0;
+const MAX_VALUE_PRICE = 100000;
+
+const getSlider = (priceHouse, typeHouse, objectHouse) => {
+  noUiSlider.create(slider, {
+    range: {
+      min: MIN_VALUE_PRICE,
+      max: MAX_VALUE_PRICE,
+    },
+    start: 5000,
+    step: 1.0,
+    connect: 'lower',
+    format: {
+      to: function (value) {
```

Comment:
```suggestion
      to: (value) => {
```
ты везде используешь стрелочные функции. давай и тут их тоже использовать)